### ISSUE TITLE
Add common used Tags For faster Tagging

### ISSUE THREAD

**Issue by ChillLP** [jheubuch/traewelling-android#395]
When using Tags, i mostly travel with the Same Ticket, the smae Trains and so on. Id would be a nice enhancement when commonly used Tags would be stored locally and suggested with the checkin.

Eg; im currently travelling with interrail and i dont Always want to Type "Interrail" or more specifically "Interrail First class youth Promotion Ticket" - If i Chose the Tag "Ticket" and i get suggestions of my Last i dont know, 5 Ticket Tags that could enhance User experience.

TL:DR Store "Tag" Values according to Keys Locally and suggest the Last 5 used ones to the user

**Comment by jheubuch** [comment]
duplicates #298

**Comment by ChillLP** [comment]
Whoops
